src/webots/scene_tree/WbInsertExternProtoDialog.cpp

```diff
@@ -68,6 +68,10 @@ WbInsertExternProtoDialog::WbInsertExternProtoDialog(QWidget *parent) : mRetriev
 }
 
 WbInsertExternProtoDialog::~WbInsertExternProtoDialog() {
+  delete mTree;
+  delete mSearchBar;
+  delete mInsertButton;
+  delete mCancelButton;
```

Comment:
Qt is already handling the destruction of the objects if you pass a QObject as parent in the constructor.
So these four delete instructions should be removed.
```suggestion
```

Reply:
Yes true, these are false positives.